### DISCUSSION TITLE
Start planning for Mui v5?

### DISCUSSION THREAD

**Discussion by nspaeth** [lookfirst/mui-rff#360] [General]
Mui version 5 is still in beta, but it might make sense to start planning for it now, or to start a new branch tracking those changes.

But first, maybe it makes sense to compile a list of all of the changes that are relevant to mui-rff as a todo list?

**Comment by lookfirst** [comment]
Sounds good. But I have zero time for that myself right now. =) Deep in golang on another project. Happy to support as necessary.

**Comment by lookfirst** [comment]
Tracking in this issue: https://github.com/lookfirst/mui-rff/issues/401
